src/Sulu/Bundle/CategoryBundle/Tests/Functional/Controller/CategoryControllerTest.php

```diff
@@ -173,7 +173,7 @@ public function testGetById()
 
         $client->request(
             'GET',
-            '/api/categories/' . $this->category1->getId()
+            '/api/categories/' . $this->category1->getId() . '?locale=en'
```

Comment:
`locale` is a mandatory parameter now, right? Can we also add a test which checks if the request fails, when the locale is not passed?